lips/lip-9.md

```diff
@@ -0,0 +1,95 @@
+---
+lip: 9
+title: Libra Private Transaction Metadata
+authors: Daniel Prinz
+status: Draft
+type: Informational
+created: 09/10/2020
+---
+# Summary
+This LIP describes the conceptual model and implementation of a private, on-chain description attribute, that could be added to every P2P transaction. An example of that would be short and descriptive information about the transaction, like what are the purchased items that are being paid in a transaction. 
+
+# Abstract / Overview
+The Libra blockchain can carry informational metadata with every transaction made. This data could vary and usually used for validating both sides of the transaction could "understand" its mean. While this metadata deals with the functioning of the sender and receiver VASPs, and/or with the FATF travel rule regulatory constraints, the transaction info itself remains opaque to the end-users. This property is missing from the current payment experience of any traditional network.
```

Comment:
This paragraph is unclear to me.  I'm not entirely sure what it's trying to convey

Reply:
It meant to explain why do we need such details on-chain. 
currently, transaction metadata carrying only functional data (like travel rule reference, subaddresses, etc), but I think it should support an extra level of private data.
In legacy payment systems, the issuer usually can not present the user with great details about a specific transaction, rather some generic idea. Think of a credit card statement, in which you can see that you paid to Amazon, but you can not tell what you've bought. For that, you need to login into your account at Amazon and look for these details. What if each transaction of your wallet was carrying extra data that describes the payment in the very same way (and details) as you saw it when authorizing the payment? it would give you as a user a lot more power and control over your account. Also, it opens a new section of fin-apps to analyze your purchases on your behalf and make optimization suggestions for your benefit.